bids/modeling/statsmodels.py

```diff
@@ -279,13 +280,7 @@ def __init__(self, level, name, transformations=None, model=None,
         self.children = []
         self.parents = []
         if group_by is None:
-            group_by = []
-            # Loop over contrasts after first level
-            if self.level != "run":
-                group_by.append("contrast")
-            # Loop over node level of this node
-            if self.level != "dataset":
-                group_by.append(self.level)
+            raise ValueError(f"group_by is not defined for Node: {name}")
```

Comment:
We should not be handling if `group_by` is None, because it should always be defined